pkg/cloudprovider/types.go

```diff
@@ -85,6 +85,28 @@ type CapacityConstraints struct {
 // CapacityPacking is a solution to packing pods onto nodes given constraints.
 type CapacityPacking map[*v1.Node][]*v1.Pod
 
+// Packings contains the result of packing decision made by a CapacityPacking, an
+// instance ID as key and value is the node object and the pods packed in the instance
+type Packings map[string]*NodePacking
```

Comment:
I'm not sure I'm sold on exposing this at the cloud provider level. InstanceTypeOptions, etc seems like it's leaking our implementation for how we're computing the packing. At the cloud provider level, I think the original CapacityPacking struct is the correct abstraction.

Reply:
I could imagine that a packing is a 
```
struct {
  Node
  Pods
}
```
But I think Id and InstanceTypeOptions are both awkward.